global_variable_generator.py

```diff
@@ -400,15 +412,18 @@ def calc_addr_offset_hmi_tag(is_array: bool, var_type: str, offset: str) -> Unio
         return (0.1) if is_array else float(offset)
     elif var_type == "WORD":
         return (1) if is_array else int(offset)
+    elif var_type == "INT":

```

Comment:
Given it some thoughts, I think it would be better if we forbid "INT" type in the template.

Could you change the code and template accordingly? So that template type should either be "BOOL" or "WORD". "BIT" in HMI tab is an exception, we disregard it for now.

Reply:
Those that are "INT", let's change it to "WORD" instead.